website/docs/Use-Cases/enhanced_inference.md

```diff
@@ -165,77 +165,6 @@ Total cost: 0.00027
 
 Note: if using a custom model client (see [here](/blog/2024/01/26/Custom-Models) for details) and if usage summary is not implemented, then the usage summary will not be available.
 
-## Caching
```

Comment:
Can we keep the section title and a link to the new page? That is safer in case we have other pointers or hints that people would expect a section here.